pom.xml

```diff
@@ -582,8 +582,8 @@
 			<!-- openSaml -->
 			<url>https://build.shibboleth.net/maven/releases/</url>
 		</repository>
-		<!-- <repository> <id>uwdigi-repo-central</id> <name>DIGI Public Repository</name>
-			<url>https://packages.uwdigi.org/artifactory/public</url> </repository> -->
+		<repository> <id>uwdigi-repo-central</id> <name>DIGI Public Repository</name>
```

Comment:
This service is currebtly down . and the solution is to build the data export depenedece locally. see https://github.com/I-TECH-UW/OpenELIS-Global-2?tab=readme-ov-file#running-openelis-global2-using-docker-compose-with-locally-compiledbuilt-artifacts-ie-the-war-file-and-react-code